src/app/technical-record/authorisation-into-service/authorisation-into-service.component.html

```diff
@@ -3,7 +3,7 @@
     <dt class="govuk-summary-list__key">COC issue date</dt>
     <dd id="test-authIntoService-cocIssueDate" class="govuk-summary-list__value">
       <span *ngIf="authoIntoService?.cocIssueDate; else hyphen">
-        {{ authoIntoService?.cocIssueDate | date: 'dd/MM/yyyy' }}
+        {{ authoIntoService?.cocIssueDate | date: 'dd/MM/yyyy':'UTC' }}
```

Comment:
please revert `"authIntoService": {
          "cocIssueDate": "2020-05-15",
          "dateReceived": "2020-05-15",
          "datePending": "2020-05-15",
          "dateAuthorised": "2020-05-15",
          "dateRejected": "2020-05-15"
        }`

 As per the API